src/Shared/ToolsetElement.cs

```diff
@@ -306,7 +303,7 @@ public PropertyElementCollection PropertyElements
         /// </summary>
         internal sealed class PropertyElementCollection : ConfigurationElementCollection
         {
-            #region Private Fields
+    #region Private Fields
```

Comment:
It looks like the convention, at least in MSBuild, is to align the `#region` / `#endregion` directives same as the enclosed code, so this change is likely not desired.